lib/utils/SchemaBuilderXSD.js

```diff
@@ -681,7 +682,7 @@ class SchemaBuilderXSD {
       wsdlElement.pattern = element.pattern;
       wsdlElement.enum = element.enum;
 
-      this.getChildren(element, wsdlElement, elements);
+      this.getChildren(getDeepCopyOfObject(element), wsdlElement, elements, targetNamespace);
```

Comment:
```suggestion
      this.getChildren(element, wsdlElement, elements, targetNamespace);
```

Reply:
@VShingala I had removed this getDeepCopyOfObject from here as part of #196, and it got added back again, I believe, due to resolving conflicts. We should remove this and test it.

Reply:
@dhwaneetbhatt Sure, I'll take a look again and see if we are using it elsewhere where not needed.